src/Orchard.Web/Modules/Orchard.Users/Views/Register.cshtml

```diff
@@ -1,32 +1,53 @@
-﻿<h1>@Html.TitleForPage(T("Create a New Account").ToString())</h1>
+﻿@{
+    var requirementsMessage = T.Plural("The password can't be empty", "Passwords are required to be a minimum of {0} characters in length", (int)ViewData["PasswordLength"]).Text;
+    var additionalRequirementsMessage = new List<string>();
+    if ((bool)ViewData["LowercaseRequirement"]) {
+        additionalRequirementsMessage.Add(T("one lowercase letter").Text);
```

Comment:
Can you ensure the Text property is not html encoded, otherwise it would result in double encoding in the output. To try it just replace some text with html, it should be encoded only once.

Reply:
I can confirm that T("").Text does not html encode the string.
![image](https://user-images.githubusercontent.com/13333709/70778718-29f40280-1d83-11ea-8502-7e18994c3104.png)

Reply:
Looking forward to the code I noticed we need to have similar changes somewhere else.
I will integrate this PR soon.

Reply:
Done with 2 new commits